syntaxes/styled-components.json

```diff
@@ -1,6 +1,6 @@
 {
 	"fileTypes": ["js", "jsx", "ts", "tsx"],
-	"injectionSelector": "L:source -comment",
+	"injectionSelector": "L:source",
```

Comment:
This causes CSS to be highlighted inside of comments which is not the desired behaviour I believe.

It should also not be necessary for your change to work.

Reply:
@Cu3PO42  I've tried with it. ` -comment` exclude comments and RegExp for comment not working.

Reply:
`-comment` does not prevent you from consuming comments, it prevents the rule from being injected into the scope of a comment.

I just tried your branch with `-comment` and it works as expected.